main.tex

```diff
@@ -72,7 +77,7 @@
 
 
 \title{NoSQL Databases}
-\author{Daniel Rutz and Leon Schürmann (Eds.)}
+\author{Class StgInf16A of DHBW Stuttgart}
```

Comment:
If you insist on changing this, please spell the course abbreviation correctly:
```
Course STG-TINF16A of the Baden-Wuerttemberg Cooperative State University Stuttgart
```

As seen [here](https://rapla.dhbw-stuttgart.de/rapla?page=calendarlist) and [here](https://www.dhbw-stuttgart.de/zielgruppen/international-visitors/overview/)

Reply:
I just hope it doesn't look bad when it's that long. That's why I wanted to keep it short.